README.md

```diff
@@ -2,30 +2,139 @@
 
 ## Getting Started
 
-OpenTreeMap for iOS has 2 major dependencies:
+### Service Dependencies
 
-  - An OpenTreeMap web application installation
-  - A "skin," which is a set of images and configurations files
+OpenTreeMap for iOS has 2 major web service dependencies:
 
-The source for the OpenTreeMap web application is available on github
+  - An OpenTreeMap web application and ecobenefits service installation
+  - An OpenTreeMap tiler installation
 
-<a href="https://github.com/OpenTreeMap/otm-core">https://github.com/OpenTreeMap/otm-core</a>
+The source code for these services is available is available on Github:
 
-A default skin can be downloaded using the Fabric (http://docs.fabfile.org/) script included with the OpenTreeMap for iOS source.
+- https://github.com/OpenTreeMap/otm-core
+- https://github.com/OpenTreeMap/otm-ecoservice
+- https://github.com/OpenTreeMap/otm-tiler
+
+
+### Configuration files
```

Comment:
Please consider this update. (note - listing entitlements file first so it doesn't disappear after the long variable list.)

The iOS application requires three configuration files in the `OpenTreeMap` directory:
* `OpenTreeMap.entitlements`
* `OpenTreeMap-Info.plist`
* `Implementation.plist`

The `OpenTreeMap` directory contains templates for all three, and a Fabric command for generating `OpenTreeMap-Info.plist`.

#### OpenTreeMap.entitlements